### PR TITLE
[DOCS-7059] Add transactional information

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: process-automation/latest/admin/release.md [hunk 1]
```diff
@@ -197,7 +197,7 @@ You will see the Monitoring dashboard for the application. Services that are ope
 
 ### Upgrade
 
-Upgrading an application allows for a new version of a released project to be deployed to an existing application. Tasks and process instances that are in progress and based on a previous application version can still be completed, however any new ones started will use the new model definitions.
+Upgrading an application allows for a new version of a released project to be deployed to an existing application. Tasks and process instances that are in progress and based on a previous application version can still be completed, however any new ones started will use the new model definitions. The upgrade process is transactional, which means that if there is an error during the upgrade process the application is automatically rolled back to the previous stable version.
```
Comment (miguelruizdev) [inline process-automation/latest/admin/release.md:200]: Didn't we add the new text as a note, and moved the old note section to be part of the main paragrahp?

**Review by miguelruizdev — CHANGES_REQUESTED** [review]

File: process-automation/latest/admin/release.md (hunk 1 above)
Comment (oliveralfresco2018) [reply process-automation/latest/admin/release.md]: Hi, yeah its there I think we're too quick for github. Take a look now! :-)

**Review by miguelruizdev — APPROVED** [review]
LGTM

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**